lib/jsdom/level1/core.js

```diff
@@ -245,7 +245,7 @@ core.DOMImplementation.prototype = {
 };
 
 core.Node = function Node(ownerDocument) {
-  EventTarget.call(this);
+  EventTarget.init.call(this);
```

Comment:
Basically I am against trying to use things for purposes they were not meant for. The init stuff is clearly neither a constructor nor a method. We shouldn't use the language's built-in method and constructor functionality for that.

E.g. if I were to do the prototype approach for impls, then I would just declare prototype objects and not have them be attached to classes.